tests/types/index.ts

```diff
@@ -172,6 +179,13 @@ const retrievedIbanElement: StripeIbanElement | null = elements.getElement(
   'iban'
 );
 
+// @ts-expect-error: `extra_property` is not valid
```

Comment:
Will this work with previous versions we're still testing against ?

Reply:
It passes CI.

Reply:
The previous versions we test against aren't typechecking `tests/types/index.ts`, but rather a much smaller example app. I bet we can fix that if we split out the tests into two files—one that uses `@ts-expect-error`, and one that doesn't.